package.json

```diff
@@ -41,7 +41,7 @@
     "example": "node ./bin/tldr tar",
     "test": "mocha test --require=env-test",
     "test:quiet": "npm test --reporter=dot",
-    "lint": "eslint lib test bin/tldr",
+    "lint": "eslint --global Promise lib test bin/tldr",
```

Comment:
Please set this in the `.eslintrc`. Avoid putting settings from the command line.

Reply:
Taken care of in #166. You can remove this.

Reply:
Apologies for this. Didn't notice the `.eslintrc` file earlier.